examples/dynamic-form-element-options/README.md

```diff
@@ -7,3 +7,5 @@
 -   [`GET /jamaica-100m-relay`](./src/jamaica-100m-relay.js)
 
 -   [`GET /yes-no-maybe`](./src/yes-no-maybe.js)
+
+-   [`GET /airports`](./src/airports.js)
```

Comment:
Do you think its worth add a little description to each of these outlining the features they demonstrate? E.g. this one has attributes, the one above has colours (I assume).

Prob also worth working with Blake to add links for further docs/info to support portal.